packages/react-devtools-core/src/backend.js

```diff
@@ -138,6 +138,7 @@ function setupBackend(wall, resolveRNStyle) {
   var bridge = new Bridge(wall);
   var agent = new Agent(window, {
     rnStyle: !!resolveRNStyle,
+    rnStyleMeasure: !!resolveRNStyle,
```

Comment:
I added [`measureSupport`](https://github.com/jhen0409/react-devtools/blob/4a332c2e9becadb3d2774f5971b09f836b88ec4b/plugins/ReactNativeStyle/ReactNativeStyle.js#L57) prop in NativeStyler (from [`store.capabilities.rnStyleMeasure`](https://github.com/jhen0409/react-devtools/blob/4a332c2e9becadb3d2774f5971b09f836b88ec4b/frontend/Panel.js#L242)) to backward compatibility for [the backend vendor](https://github.com/facebook/react-devtools/tree/master/packages/react-devtools-core/vendor).